ledger-core/application/testutils/launchnet/launchnet.go

```diff
@@ -370,7 +370,13 @@ func startNet() (*exec.Cmd, error) {
 		_ = os.Chdir(cwd)
 	}()
 
-	cmd := exec.Command("./scripts/insolard/launchnet.sh", "-pwdg")
+	args := "-pwdg"
+	cloudMode := os.Getenv("CLOUD_MODE")
+	if strings.ToLower(cloudMode) == "true" {
+		args += "m"
```

Comment:
m?

Reply:
if it is cloud mode - then "c" would probably suit better

Reply:
As for me - 'c' - for configs. That is why I used 'm'